R/make_project.R

```diff
@@ -65,15 +65,13 @@ make_project <- function(
   
   # get version of Quarto on the machine and save it as a version
   the_version <- quarto::quarto_version()
-  
-  
   if (vignette == FALSE){ # make paper project w/o package infrastructure
     # If the project object does not exist add it.
     if (length(list.files(path = path, pattern = "\\.Rproj$")) == 0) {
       usethis::create_project(path = path, open = TRUE, rstudio = TRUE)
     }
   } else { # make paper project with package infrastructure
-    if (type == "Quarto (analysis.qmd)" & the_version < "2.4.549"){
+    if (type == "Quarto (analysis.qmd)" & the_version < "1.4.549"){
```

Comment:
good catch! i'm sorry i missed that earlier

Reply:
Please add a comment with URL to the NEWS file or release notes detailing *why* this particular version is the cutoff. Here are some release notes; it's probably in here somewhere: <https://quarto.org/docs/download/release.html>